src/parsec_client/operations/can_do_crypto.md

```diff
@@ -0,0 +1,58 @@
+# CanDoCrypto
+
+Check if the provider supports:
+
+- using a specific algorithm with an existing key
+- generating a key and optionally using it for a specific algorithm
+- importing a key and optionally using it for a specific algorithm
+- deriving a key and optionally using it for a specific algorithm (to be checked)
+
+Opcode: 32 (`0x0020`)
+
+**(EXPERIMENTAL) The parameters for key attestation are in an experimental phase. No guarantees are
+offered around the stability of the interface for any key attestation mechanism.**
```

Comment:
Bit of copy-pasta here 😀

Reply:
Oops yeah I forgot to edit it

Reply:
How does this sound instead?
`**(EXPERIMENTAL) This operation is still being implemented and so no guarantees are
offered around the stability of the interface for any key attestation mechanism.**`

Reply:
Hmm, probably something pointing towards "capability discovery" instead of "key attestation" would be clearer!